syntax = "proto3";

package flyteidl.admin;
option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin";


// Encapsulates specifications for routing an execution onto a specific cluster.
message ClusterAssignment {
  reserved 1, 2;
  string cluster_pool_name = 3;
}
